Sources/Commands/Describe.swift

```diff
@@ -100,16 +100,16 @@ fileprivate struct DescribedPackage: Encodable {
     
     /// Represents a package dependency for the sole purpose of generating a description.
     enum DescribedPackageDependency: Encodable {
```

Comment:
having the identity when using `package describe` is extremely useful when you are not sure what package name to put in the target dependency package name

Reply:
Absolutely, this makes a ton of sense.